pai-management/bootstrap/grafana/grafana-configuration/pai-clusterview-dashboard.json.template

```diff
@@ -388,7 +388,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "100 - avg (irate(node_cpu_seconds_total{mode=\"idle\"}[5m])) * 100",
+              "expr": "100 - avg (irate(node_cpu_seconds_total{mode=\"idle\"}[{{clusterinfo['prometheusinfo']['scrape_interval']}}0s])) * 100",
```

Comment:
We can follow the mock clusterinfo currently, but should refactor it in future with formal solutions.

Reply:
Remind by Di, the Jinja support arithmetic, so in webportal fix, I changed it to " * 10 ", I will change the grafana settings in this way as well.